homeassistant/components/frank_energie/config_flow.py

```diff
@@ -0,0 +1,112 @@
+"""Config flow for Picnic integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from python_frank_energie import FrankEnergie
+from python_frank_energie.exceptions import AuthException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_AUTHENTICATION, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_AUTH_TOKEN, CONF_REFRESH_TOKEN, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle the config flow for Frank Energie."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self._reauth_entry: ConfigEntry | None = None
```

Comment:
Can we add reauth in a follow up PR?

Reply:
Removed Reauth handling, token renewal is still included as the token expires after 7 days.